src/js/common/services/device.service.js

```diff
@@ -1,5 +1,28 @@
 import { protectAPI } from 'APIs';
 
+/**
+  Add these attrs below when Backstage is ready:
```

Comment:
@luisfelipefurlan This is my suggestion for the Backstage structure. I commented out this code to avoid breaking things, so this PR can be safely merged. In other mutations I believe I don't need the returned data as the user is redirected to the device list.